package.json

```diff
@@ -23,8 +23,12 @@
     "@babel/preset-react": "^7.16.5",
     "@babel/preset-typescript": "^7.16.5",
     "@babel/runtime": "^7.16.5",
+    "@date-io/date-fns": "^1.3.13",
     "@emotion/react": "^11.7.1",
     "@mui/icons-material": "^5.2.5",
+    "@mui/material": "^5.8.2",
+    "@mui/styles": "^5.8.0",
+    "@mui/x-date-pickers": "^5.0.0-alpha.5",
```

Comment:
All of the three new dependencies are unfortunately required for `@mui/x-date-pickers`. We'll want to switch this if we get a pro license, though. 😢